### PR TITLE
CPr NGSIv2 forwarding based in POST /v2/op/query

### PR DESCRIPTION
**Pull request by fgalan**
The adaptation has been pretty easy, thanks to previous modifications by @kzangeli. Only some renders needed to be added.

### CODE REVIEW

File: doc/manuals/devel/cprs.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ The `QueryContextRequest` items are filled in based on the output of the [**mong
 _FW-04: `queryForward()` function detail_
 
 * Parse the context provider string to extract IP, port, URI path, etc. (step 1).
-* The request to forward has to be built (step 2). In the case of NGSIv1, we need to extract information of the binary object into text to be able to send the REST request (plain text) to the Context Provider using POST /v1/queryContext. In the case of NGSIv2, the use GET /v2/entities and no payload is required.
+* The request to forward has to be built (step 2). In the case of NGSIv1, we need to extract information of the binary object into text to be able to send the REST request (plain text) to the Context Provider using POST /v1/queryContext. In the case of NGSIv2, the use POST /v2/op/query is required.
```
Comment (fgalan) [inline doc/manuals/devel/cprs.md]: A deeper doc review needs to be done before merging (a couple of png figures need to be updated at least).

**Comment by jmcanterafonseca** [comment]
there is currently no agreement in doing things this way and we should be awaiting for a TSC decision ...  

/cc @fgalan @kzangeli

**Comment by fgalan** [comment]
Feedback from TSC (or from anyone! :) would be of course very useful in this PR, but the lack of it will not block its progress. If it doesn't arrive on time then please use https://github.com/telefonicaid/fiware-orion/issues/3485 (in someway a "continuation issue" for this) to provide it. Thanks!

**Comment by fgalan** [comment]
CC: @fisuda, as this this PR includes some modification to .md files that should be synchornized in the Japanese translation.

**Comment by fisuda** [comment]
Noted with thanks.

**Pull request merged**